### DISCUSSION TITLE
Full Project Diagnostics

### DISCUSSION THREAD

**Discussion by josephcrawfordSRH** [LazyVim/LazyVim#4852] [Q&A]
Is there something I can do so that when I do `<leader>xx` it will show all diagnostics across my entire project?  I am working on a typescript project and I cannot see anything except for the errors in the open buffer.  I have disabled tabs so I only have a single buffer open at a time unless I have splits.

It is a bit cumbersome to not see all errors in one place so that I can move around and fix them.

**Comment by Daydreamer-riri** [comment]
I use TSC.nvim。But I also want to know if there is a better solution. Because I still need to deal with ESLint

**Comment by josephcrawfordSRH** [comment]
@Daydreamer-riri for now I am using `artemave/workspace-diagnostics.nvim` in order to get it to work with typescript files, I have not tested other languages but I would be shocked if it was not supported by default somehow..

**Reply by Daydreamer-riri** [reply]
Thanks. This plugin is great, but I found I couldn't get it to work with eslint.
